modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/EnrollmentService.java

```diff
@@ -97,9 +75,12 @@ public Message isEnrolled(@PathParam("id") String id) throws AndroidAgentExcepti
 			result = AndroidAPIUtils.getDeviceManagementService().isEnrolled(deviceIdentifier);
 			if (result) {
 				responseMsg.setResponseMessage("Device has already enrolled");
+				responseMsg.setResponseCode(Response.Status.ACCEPTED.toString());
```

Comment:
The reason that 'toString()' method is used because the 'responseMsg' object is an instance of Message class (https://github.com/wso2/product-mdm/blob/master/modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/util/Message.java). It has the 'responseCode' defined as the String type. That is why this has been converted into a string. 
I think its better if we have used 'responseCode' attribute as Status type in the Message class.